docs/user_guide/workloads/run_spark.md

```diff
@@ -25,12 +25,86 @@ specific language governing permissions and limitations
 under the License.
 -->
 
+## Deploy Spark job with Spark Operator and Helm
+
+:::note
+Pre-requisites:
+- This tutorial assumes YuniKorn is [installed](../../get_started/get_started.md) under the namespace `yunikorn`
+- Use spark-operator version >= 2.0 to enable support for YuniKorn gang scheduling
+:::
+
+
+:::warning
+This installation involves installing YuniKorn and Spark operator, which may take a few minutes to complete. To check the status we can use `kubectl get pods -n yunikorn` and `kubectl get pods -n spark-operator`
+:::
+
+### Install YuniKorn
+A simple script to install YuniKorn under the namespace `yunikorn`, refer to [Get Started](../../get_started/get_started.md) for more details.
+```shell script
+helm upgrade --install yunikorn yunikorn/yunikorn \
+  --create-namespace \
+  --namespace yunikorn
+```
+
+### Install spark operator
+
+We should install `spark-operator` with `controller.batchScheduler.enable=true` and set `controller.batchScheduler.default=yunikorn`. It's optional to set the default scheduler to YuniKorn since you can specify it later on, but it's recommended to do so.
+
+```shell script
+helm repo update # (optional) update the helm repo cache to prvent install wrong version
+```
+
+```shell script
+helm upgrade --install spark-operator spark-operator/spark-operator \
+  --create-namespace \
+  --namespace spark-operator \
+  --set controller.batchScheduler.enable=true \
+  --set controller.batchScheduler.default=yunikorn
+```
```

Comment:
I prefer to put all in one block. Similar to something in RayOperator setup:
- https://yunikorn.apache.org/docs/user_guide/workloads/run_ray_cluster#setup-a-kuberay-operator

```shell script
helm repo add spark-operator https://kubeflow.github.io/spark-operator
helm repo update 
helm upgrade --install spark-operator spark-operator/spark-operator \
  --create-namespace \
  --namespace spark-operator \
  --set controller.batchScheduler.enable=true \
  --set controller.batchScheduler.default=yunikorn
```

No need to add the 'optional' phrase.
And could we change the upgrade `helm upgrade` to `helm install`?  We are trying to install spark-operator here.

Reply:
Hi @chenyulin0719, I get your point. The best practice indeed is:
```
helm repo add spark-operator https://kubeflow.github.io/spark-operator
helm repo update 
helm install spark-operator spark-operator/spark-operator \
  --create-namespace \
  --namespace spark-operator \
  --set controller.batchScheduler.enable=true \
  --set controller.batchScheduler.default=yunikorn
```